app/packs/review/page.tsx

```diff
@@ -12,24 +13,20 @@ export const metadata: Metadata = {
 async function Review() {
 	const packs = await prisma.questionPack.findMany({
 		where: {
-			OR: [{ pending: true }, { denied: true }]
+			OR: [{ status: Status.pending }, { status: Status.pending }]
```

Comment:
You are checking for the pending status twice, potentially you intended to check for pending & resubmit_pending
```suggestion
			OR: [{ status: Status.pending }, { status: Status.resubmit_pending }]
```